src/app/components/alertifier.js

```diff
@@ -11,7 +11,7 @@ export const SALUTATION = 'Hey!';
  *
  * Usage: const myAlertifier = Alertifier(document.querySelector('#myAlertifier'));
  */
-export default Alertifier = (container) => {
+export default (container) => {
```

Comment:
I think as the function is not named, it could be printed out as an anonymous function in the stack trace…
`export default const Alertifier = …`?